core/spec.md

```diff
@@ -4035,9 +4067,12 @@ and the following Resource level attributes:
   - `none` - No compatibility checking is performed.
 
 - Constraints:
-  - If present, it MUST be a case sensitive value from the model defined
+  - If present, it MUST be a case-sensitive value from the model defined
     enumeration range.
   - When not specified, the default value MUST be `none`.
+  - The value MAY change only when the new `compatibility` value can be
+    enforced across all existing versions. If that's not the case, the
```

Comment:
s/versions/Versions/

s/refuse the change/generate an error/   <- I'm trying to use this phrase everywhere so I can easily add a reference to the error we expect people to use.